Controllers/users.php

```diff
@@ -4,4 +4,17 @@ protected function register(){
 		$viewmodel = new UserModel();
 		$this->returnView($viewmodel->register(), true);
 	}
+	protected function login(){
+		$viewmodel = new UserModel();
+		$this->returnView($viewmodel->login(), true);
+}
+
+
+protected function logout(){
+	unset($_SESSION['is_logged_in']);
+	unset($_SESSION['user_data']);
```

Comment:
You can chain multiple key to unset in one line
```suggestion
	unset($_SESSION['is_logged_in'], $_SESSION['user_data']);
```